### ISSUE TITLE
Update from specification to definition

### ISSUE THREAD

**Issue by benstoltz** [Esri/cedar#393]
There are certain places (mainly in cedar-amcharts) that reference or use the word specification or spec instead of definition. Since we've made a distinction between the two we should fix this.

@tomwayson thoughts?

For example in cedar: 
```
  public render() {
    cedarAmCharts(this._container, this.definition(), this.data())
    return this
  }
```

While in cedar-amcharts it comes in like:
```
export function cedarAmCharts(elementId: string, spec: any, data?: any[]) {
  if ((!elementId || !spec || !data) && (spec.type && spec.type !== 'custom')) {
    const err = new Error('Element Id, specification, and data are all required.')
    throw err
  }

  if (spec.type && spec.type === 'custom') {
    return render.renderChart(elementId, spec)
  }
  return render.renderChart(elementId, spec, data)
}
```

**Comment by tomwayson** [comment]
Yes! 💯

**Comment by benstoltz** [comment]
Was resolved in #399
